<launch>

  <arg name="robot_name"                        default="locobot"/>
  <arg name="rviz_frame"                        default="$(arg robot_name)/odom"/>
  <arg name="rvizconfig"                        default="$(find locobot_custom)/rviz/locobot_move_base+move_it.rviz" />

  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-f $(arg rviz_frame) -d $(arg rvizconfig)"
    ns="$(arg robot_name)">
    <remap from="/clicked_point" to="clicked_point"/>
    <remap from="/initialpose" to="initialpose"/>
    <remap from="/move_base_simple/goal" to="move_base_simple/goal"/>
  </node>

</launch>
